diff --git a/client/src/App.jsx b/client/src/App.jsx
index 1f6acc9..63a9189 100644
--- a/client/src/App.jsx
+++ b/client/src/App.jsx
@@ -23,48 +23,50 @@ const App = () => {
 	const isEducatorRoute = useMatch("/educator/*");
 
 	return (
-		<div className="text-default min-h-screen bg-white">
+		<div className="text-default min-h-screen bg-white flex flex-col">
 			<ToastContainer />
 			{!isEducatorRoute && <Navbar />}
-			<Routes>
-				<Route path="/" element={<Home />} />
-				<Route path="/course-list" element={<CoursesList />} />
-				<Route path="/course-list/:input" element={<CoursesList />} />
-				<Route path="/course/:id" element={<CourseDetails />} />
-				<Route path="/login" element={<Login />} />
-				<Route path="/register" element={<Register />} />
-				<Route
-					path="/my-enrollments"
-					element={
-						<ProtectedRoute>
-							<MyEnrollments />
-						</ProtectedRoute>
-					}
-				/>
-				<Route
-					path="/player/:courseId"
-					element={
-						<ProtectedRoute>
-							<Player />
-						</ProtectedRoute>
-					}
-				/>
-				<Route path="/loading/:path" element={<Loading />} />
+			<main className="flex-1">
+				<Routes>
+					<Route path="/" element={<Home />} />
+					<Route path="/course-list" element={<CoursesList />} />
+					<Route path="/course-list/:input" element={<CoursesList />} />
+					<Route path="/course/:id" element={<CourseDetails />} />
+					<Route path="/login" element={<Login />} />
+					<Route path="/register" element={<Register />} />
+					<Route
+						path="/my-enrollments"
+						element={
+							<ProtectedRoute>
+								<MyEnrollments />
+							</ProtectedRoute>
+						}
+					/>
+					<Route
+						path="/player/:courseId"
+						element={
+							<ProtectedRoute>
+								<Player />
+							</ProtectedRoute>
+						}
+					/>
+					<Route path="/loading/:path" element={<Loading />} />
 
-				<Route
-					path="/educator"
-					element={
-						<ProtectedRoute requireEducator>
-							<Educator />
-						</ProtectedRoute>
-					}
-				>
-					<Route index element={<Dashboard />} />
-					<Route path="add-course" element={<AddCourse />} />
-					<Route path="my-courses" element={<MyCourses />} />
-					<Route path="student-enrolled" element={<StudentsEnrolled />} />
-				</Route>
-			</Routes>
+					<Route
+						path="/educator"
+						element={
+							<ProtectedRoute requireEducator>
+								<Educator />
+							</ProtectedRoute>
+						}
+					>
+						<Route index element={<Dashboard />} />
+						<Route path="add-course" element={<AddCourse />} />
+						<Route path="my-courses" element={<MyCourses />} />
+						<Route path="student-enrolled" element={<StudentsEnrolled />} />
+					</Route>
+				</Routes>
+			</main>
 			{!isEducatorRoute && <Footer />}
 		</div>
 	);
diff --git a/client/src/pages/educator/Educator.jsx b/client/src/pages/educator/Educator.jsx
index 39258d6..6566118 100644
--- a/client/src/pages/educator/Educator.jsx
+++ b/client/src/pages/educator/Educator.jsx
@@ -6,9 +6,9 @@ import Footer from "../../components/educator/Footer";
 
 const Educator = () => {
 	return (
-		<div className="text-default min-h-screen bg-white transition-colors">
+		<div className="text-default min-h-screen bg-white transition-colors flex flex-col">
 			<Navbar />
-			<div className="flex">
+			<div className="flex flex-1">
 				<Sidebar />
 				<div className="flex-1">{<Outlet />}</div>
 			</div>
